test/studies/shootout/submitted/knucleotide3.chpl

```diff
@@ -22,17 +22,21 @@ proc main(args: [] string) {
       lineSize = 0,
       numRead = 0;
 
-  while stdinNoLock.readline(buff, lineSize) && !startsWithThree(buff) do
+  lineSize = stdinNoLock.readLine(buff);
+  while lineSize>0 && !startsWithThree(buff) {
     numRead += lineSize;
-
+    lineSize = stdinNoLock.readLine(buff);
+  }
   // Read in the rest of the file
   var dataDom = {1..fileLen-numRead},
       data: [dataDom] uint(8),
       idx = 1;
 
-  while stdinNoLock.readline(data, lineSize, idx) do
+  lineSize = stdinNoLock.readLine(data[idx..]);
+  while lineSize>0 {
     idx += lineSize - 1;
-
+    lineSize = stdinNoLock.readLine(data[idx..]);
+  }
```

Comment:
This is a common issue (so I'm mentioning it more for @mppf's sake as a reviewer than Shreyas's), but we try to keep this `shootout/submitted/` directory in sync with what's live on the CLBG site so that at any given time we know how our performance or behavior is changing without submitting new code to the site.  In this case, I'd have kept the code the same and added the deprecation warning to the .good file (then, after 1.27 is released, we can opt to submit a new version to the site that switches routines to avoid the warning; but we can't submit that code to the site before then since it wouldn't work with 1.26).

[Michael might say "it'd be nice if we could script that check as a pre-merge PR step" and I'd agree.]

Also, since we're evaluated based on code size, this would definitely be a case to, ultimately, avoid the duplication for.

Reply:
Yeah I did miss that one thanks for pointing it out Brad

Reply:
A PR against our repo you mean?  One of us should...  I can get to it eventually if Shreyas is feeling stressed about getting back to SHOC.

Reply:
If its to revert the files under the `shootout` directory to their old state and simple update the `.good` files I can do that.

Reply:
Yeah, that's the proposal—thanks!

Reply:
(you can run ./clbg-diff.py in that directory to verify that things match the site).